driver/compile.go

```diff
@@ -107,6 +107,45 @@ func compileSingle(ctx context.Context, program ast.Proc, zctx *resolver.Context
 	return newMuxOutput(pctx, leaves, sn), nil
 }
 
+func compileParallel(ctx context.Context, program ast.Proc, zctx *resolver.Context, readers []zbuf.Reader, cfg Config) (*muxOutput, error) {
```

Comment:
Are 
```go
compileSingle(ctx, program, zctx, reader, cfg)
```
and
```go
compileParallel(ctx, program, zctx, []zbuf.Reader{reader}, cfg)
```
equivalent?

If so, I think we should just replace compileSingle with this.